features/fout-meldingen.feature

```diff
@@ -0,0 +1,168 @@
+        Als '/wozobjecten?rsin=' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                         |
+        | title  | Een of meerdere parameters zijn niet correct.  |
+        | status | 400                                            |
+        | detail | Waarde voor parameter 'rsin' is niet opgegeven |
+        En bevat de response de volgende invalidParams
+        | name | reason                |
+        | rsin | geen waarde opgegeven |
+
+    Scenario: Er is geen waarde voor meerdere parameters opgegeven
+        Als '/wozobjecten?rsin=&fields=' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                    |
+        | title  | Een of meerdere parameters zijn niet correct.             |
+        | status | 400                                                       |
+        | detail | Waarde voor parameters 'rsin', 'fields' is niet opgegeven |
+        En bevat de response de volgende invalidParams
+        | name   | reason                |
+        | rsin   | geen waarde opgegeven |
+        | fields | geen waarde opgegeven |
+
+Rule: fields parameter bevat geen onbekende kenmerknamen
+
+    Abstract Scenario: Er is één of meerdere onbekende kenmerknamen opgegeven
+        Als '/wozobjecten?rsin=0345100002016017&fields=<fields waarde>' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                |
+        | title  | Een of meerdere parameters zijn niet correct.         |
+        | status | 400                                                   |
+        | detail | Parameter 'fields' bevat een (deels) ongeldige waarde |
+        En bevat de response de volgende invalidParams
+        | name   | reason                |
+        | fields | <reason omschrijving> |
+
+        Voorbeelden:
+        | fields waarde              | reason omschrijving                                          |
+        | bestaatniet                | ongeldige waarde: 'bestaatniet' opgegeven                    |
+        | bestaatniet,bestaatookniet | ongeldige waarden: 'bestaatniet', 'bestaatookniet' opgegeven |
+
+Rule: type van parameter waarde is correct
+
+    Scenario: type van waarde van een parameter is niet correct
+        Als '/wozobjecten?postcode=1234AA&huisnummer=A' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                               |
+        | title  | Een of meerdere parameters zijn niet correct.        |
+        | status | 400                                                  |
+        | detail | Parameter 'huisnummer' is niet van het correcte type |
+        En bevat de response de volgende invalidParams
+        | name       | reason                   |
+        | huisnummer | waarde 'A' is geen getal |
```

Comment:
Volgens de [common foutafhandeling.feature](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/master/features/foutafhandeling.feature) zou reason moeten zijn "Waarde is geen geldige integer."

Reply:
@MelvLee dit gaat niet om jouw experience als DX customer zero, maar om de UX?

Reply:
Dat klopt. In de description van de feature staat dat de consumer developer de foutmeldingen dan 1 op 1 kan hergebruiken in zijn UI

Reply:
:-) eeh idd!

Reply:
Naar een eindgebruiker is is "waarde 'A' is geen getal" wel vriendelijker dan "Waarde is geen geldige integer." Dus ik ben het eens met het voorstel van @MelvLee.
Het lijkt me alleen niet handig dat in hier specifiek voor WOZ te beschrijven. Kan dat niet beter aan https://github.com/VNG-Realisatie/Haal-Centraal-common/pull/111 toegevoegd worden?

Reply:
> Het lijkt me alleen niet handig dat in hier specifiek voor WOZ te beschrijven. Kan dat niet beter aan [VNG-Realisatie/Haal-Centraal-common#111](https://github.com/VNG-Realisatie/Haal-Centraal-common/pull/111) toegevoegd worden?

Lijkt mij wel. Je zou het liefst alleen WOZ-specifieke foutmeldingen in een WOZ-specifieke fout-feature zien.